web/templates/confirm_dialog/confirm_deactivate_stream.hbs

```diff
@@ -2,3 +2,18 @@
     Archiving stream <z-stream></z-stream> will immediately unsubscribe everyone. This action cannot be undone.
```

Comment:
I think this needs a wrapper as it is no longer alone here. A `<p>` tag would do the job, and then you might not need additional styling.